apps/prairielearn/src/lib/groups.sql

```diff
@@ -193,65 +188,50 @@ WITH
     WHERE
       user_id = $user_id
       AND group_id = $group_id
-  ),
-  deleted_group_user_roles AS (
-    DELETE FROM group_user_roles
-    WHERE
-      user_id = $user_id
-      AND group_id = $group_id
   )
 INSERT INTO
   group_logs (authn_user_id, user_id, group_id, action)
 VALUES
   ($authn_user_id, $user_id, $group_id, 'leave');
 
--- BLOCK reassign_group_roles_after_leave
```

Comment:
This query was virtually identical to update_group_roles, minus the log insertion, so I merged them.